src/prettytable/prettytable.py

```diff
@@ -858,9 +897,138 @@ def junction_char(self):
     @junction_char.setter
     def junction_char(self, val):
         val = str(val)
-        self._validate_option("vertical_char", val)
+        self._validate_option("junction_char", val)
         self._junction_char = val
 
+    @property
+    def top_junction_char(self):
+        """The character used when printing table borders to draw top line junctions
+
+        Arguments:
+
+        top_junction_char - single character string used to draw top line junctions"""
+        return self._top_junction_char
```

Comment:
Please could you add tests so these new properties have code coverage?

Plus there's some other uncovered bits in some stringify methods.

We can see the warnings in the files tab: https://github.com/jazzband/prettytable/pull/104/files

Reply:
Done